integration-tests/integration/dpsuser.create.spec.js

```diff
@@ -110,8 +111,29 @@ context('DPS user create functionality', () => {
     UserPage.verifyOnPage('Itag User') // stub returns this first and last name
   })
 
+  it('Should fail attempting to reach "create-user" if unauthorised', () => {
+    cy.task('stubSignIn', { roles: [{ roleCode: 'NOT_CREATE_USER' }] })
+    cy.signIn()
+    const menuPage = MenuPage.verifyOnPage()
+
+    menuPage.createDpsUserTile().should('not.exist')
+
+    cy.visit('/create-user', { failOnStatusCode: false })
+    AuthErrorPage.verifyOnPage()
+  })
+
+  it('Should fail attempting to reach "create-dps-user" if unauthorised', () => {
+    cy.task('stubSignIn', { roles: [{ roleCode: 'NOT_CREATE_USER' }] })
+    cy.signIn()
+    const menuPage = MenuPage.verifyOnPage()
+
+    menuPage.createDpsUserTile().should('not.exist')
+
+    cy.visit('/create-dps-user', { failOnStatusCode: false })
+    AuthErrorPage.verifyOnPage()
+  })
   it('Should check for CSRF token', () => {
-    cy.task('stubSignIn', { roles: [{ roleCode: 'CREATE_USER' }] })
+    cy.task('stubSignIn', { roles: [{ roleCode: 'CREATE_USER' }], tokenRoles: ['ROLE_CREATE_USER'] })
```

Comment:
roles and tokenRoles could possibly be combined, it's just that the tokenRoles is also expecting GLOBAL_SEARCH - I didn't want to change this functionality for this PR